estela-web/src/shared/header/index.tsx

```diff
@@ -38,11 +39,23 @@ export class CustomHeader extends Component<unknown> {
                             estela
                         </Link>
                     </Col>
-                    <Col flex={0.1} className="">
-                        <Notification
-                            width={26}
-                            className="hover:bg-button-hover stroke-black hover:stroke-estela rounded"
-                        />
+                    <Col flex={0.02} className="">
+                        <Dropdown
+                            overlay={
+                                <Layout className="p-5 w-96">
+                                    <NotificationsList />
```

Comment:
Notification button is pressed style when you are in notification view
<img width="800" alt="Screenshot 2022-10-27 at 10 05 07" src="https://user-images.githubusercontent.com/33165861/198326846-b766d409-155c-482e-9cda-a87a734f1610.png">